content/ansible-getting-started/3-playbook/_index.md

```diff
@@ -18,6 +18,34 @@ A Playbook can have multiple Plays and a Play can have one or multiple Tasks. In
 Here is a nice analogy: When Ansible Modules are the tools in your workshop, the Inventory is the list of materials and the Playbooks are the instructions.
 {{% /notice %}}
 
+## What are Collections and why should I care?
+
+Ansible Collections are a new distribution format for Ansible content that can include playbooks, roles, modules, and plugins. Ansible Collection names are a combination of two components. The first part is the name of the author who wrote and maintains the Ansible Collection. The second part is the name of the Ansible Collection. This allows one author to have multiple Collections. It also allows multiple authors to have Ansible Collections with the same name.
+
+    <author>.<collection>
+
+These are examples for Ansible Collection names:
+
+- ansible.posix
+
+- geerlingguy.k8s
+
+- theforeman.foreman
+
+To identify a specific module in an Ansible Collection, we add the name of it as the third part:
```

Comment:
it's not only about modules, but also roles or any plugin.

Reply:
In this case I intentionally wrote module since this part of the lab is about modules. The fact that collections can contain more than just modules has been explained in the earlier chapter.

Reply:
Add a second collection from ansible namespace as example for the concept.